forgerock-auth/src/main/java/org/forgerock/android/auth/Node.java

```diff
@@ -59,7 +62,16 @@ JSONObject toJsonObject() throws JSONException {
         }
         JSONArray array = new JSONArray();
         for (Callback cb : callbacks) {
-            array.put(new JSONObject(cb.getContent()));
+            if ((cb instanceof TextOutputCallback)) {
```

Comment:
I would handle it on receiving the Callback, 
org.forgerock.android.auth.callback.TextOutputCallback#setAttribute
Removing the content when msg type is 4

Reply:
Receiving the callback is no problem for the SDK or the server side. Submitting back is what causes issues. I am of the opinion that if we keep the callback and just remove it before submitting, we are more open to anyone using this to pass data (valuable ones not a JS script) back to the SDK. So instead of dropping the support, we just make that an non-submittable callback. Let's discuss this when you come online.